releasewarrior/cli.py

```diff
@@ -381,3 +388,57 @@ def data(push, pull, logger=LOGGER, config=CONFIG):
         git.push(logger, config)
     if pull:
         git.pull(logger, config)
+
+
+@cli.command()
+@click.argument('wnp_url')
+@click.argument('product', type=click.Choice(['firefox', 'devedition', 'fennec', 'thunderbird']))
```

Comment:
Should we just have Firefox here for now?

Reply:
Good point. I added a comment if folks want to add support for it.